<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>findForm</class>
 <widget class="QWidget" name="findForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>147</x>
     <y>110</y>
     <width>271</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Фото какого автора Вы бы хотели найти?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>140</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ФИО</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="find_text">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="find_find">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Найти</string>
   </property>
  </widget>
  <widget class="QPushButton" name="find_close">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QLabel" name="status_find">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>250</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="see_photo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Смотреть фото</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
